video_info.py

```diff
@@ -0,0 +1,60 @@
+import requests
+from datetime import datetime
+
+class VideoInfo:
+    # Where the CLI is accessing the DB, chose to access my local machine
+    def __init__(self, url="http://localhost:5000"):
+        self.url = url
+        self.selected_video = None
+        
+    # Video store employee adding new video into inventory 
+    def add_video(self, title="default video title", release_date=str(datetime.now()), total_inventory=0):
+        query_params = {
+            "title": title,
+            "release_date": release_date,
+            "total_inventory": total_inventory
+        }
+        response = requests.post(self.url+"/videos",json=query_params)
+        return response.json()
```

Comment:
One opportunity for improvement for many people for this project is handling error codes from the API requests. In general, when writing code to call an API, it's appropriate to check whether the request was successful or it triggered some sort of error. This is where HTTP response codes come in handy! We can usually assume that a 2XX response code means there was no error and any other response code means there was some sort of error. It's very common to have an if...elif...else chain to handle different ranges of response codes.

For example, if I try to add a video with an invalid inventory or date, the API will respond with an error code but no response body. This results in a JSONDecodeError when line 18 is executed. This could be avoided by printing a custom message when the response code is an error code (ie. in the range 300-500).

This applies to all the other API calls as well.